R/drive_mv.R

```diff
@@ -87,11 +93,8 @@ drive_mv <- function(file, path = NULL, name = NULL, verbose = TRUE) {
     }
   }
 
-  request <- generate_request(
```

Comment:
Note, this will affect the logic for the messaging re: renamed vs moved vs both.